server/node.go

```diff
@@ -390,6 +402,94 @@ func (n *Node) gossipCapacities() {
 	})
 }
 
+// storesScanner will walk through all the stores in the node every
+// ctx.ScanInterval and store the status in the db.
+func (n *Node) storesScanner(stopper *util.Stopper) {
+	stopper.RunWorker(func() {
+		start := time.Now()
```

Comment:
I was considering that.  Just signing up for each store's status update one at a time and as soon as all of them reported publishing the status.  But due to timings, this could mean that at worst, if everything aligned in the worst way, that the node scanner would run every scanInterval * num of stores.  So if you had 6 stores in a node, it might only update once an hour.  This way, we're not dependent on the store scanner and we'll update once every scanInterval.